skills/notion-template-processor/scripts/email_deliverer.py

```
@@ -0,0 +1,411 @@
+                recipients.extend(request.to_address)
+            else:
+                recipients.append(request.to_address)
+
+            if request.cc_addresses:
+                recipients.extend(request.cc_addresses)
+            if request.bcc_addresses:
+                recipients.extend(request.bcc_addresses)
+
+            # Send message
+            if not self.config.sender_email:
+                raise ValueError("Sender email not configured")
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)